lib/src/logic.dart

```diff
@@ -424,41 +424,47 @@ class Logic {
   }
 
   /// Accesses the [index]th bit of this signal.
+  /// Negative/Positive index values are allowed. (The negative indexing starts from where the array ends)
+  /// -([width]) <= [index] < [width]
   Logic operator [](int index) {
     return slice(index, index);
   }
 
   /// Accesses a subset of this signal from [startIndex] to [endIndex], both inclusive.
   ///
-  /// If [endIndex] is less than [startIndex], the returned value will be reversed relative
+  /// If [endIndex] comes before the [startIndex] on position, the returned value will be reversed relative
   /// to the original signal.
+  /// Negative/Positive index values are allowed. (The negative indexing starts from where the array ends)
```

Comment:
It might be good to elaborate a bit more, maybe with an example for those unfamiliar with negative indexing.  For example, check this doc comment out:
https://github.com/intel/rohd/blob/main/lib/src/values/logic_value.dart#L103-L112

Reply:
sure will work on setting an example